src/Campaigns/Blocks/blocks.ts

```diff
@@ -21,3 +21,13 @@ getAllBlocks().forEach((block) => {
         registerBlockType(block.schema as BlockConfiguration, block.settings);
     }
 });
+
+registerBlockVariation('give/donation-form', {
+    name: 'givewp-campaign-donation-form',
+    title: 'Campaign Form',
+    description: "The GiveWP Campaign Form block inserts an existing campaign form into the page.",
+    attributes: {
+        campaignId: null,
+    },
+    isActive: attributes => !!attributes.campaignId,
+});
```

Comment:
@JoshuaHungDinh looking at this, i'm not sure this is a good use case for a block variation.  Reason being, it's really just changing the name of the existing block as setting the `campaignId` to null is what the behavior will be for the existing block as the default value.

At that point, we might as well just change the name of the existing block as it should have everything it needs to function with and without a `campaignId`.